identity-service/src/routes/trackListens.js

```diff
@@ -361,4 +364,76 @@ module.exports = function (app) {
 
     return successResponse({ listenCounts: parsedListenCounts })
   }))
+
+  /*
+   * Gets the tracks and listen counts for a user.
+   * Useful for populating views like "Heavy Rotation" which
+   * require sorted lists of track listens for a given user.
+   *
+   * GET query parameters:
+   *  userId: The user id to query for
+   *  limit: (optional) The number of tracks to fetch
+   */
+  app.get('/users/listens/top', handleResponse(async (req, res) => {
```

Comment:
idk if there should be auth for this endpoint (and the next endpoint), but if you wanted we could add the authMiddleware and then get the user attached to the request for the userId.

Reply:
+1 for this and next endpoint

Reply:
great